src/components/Menu.js

```diff
@@ -34,6 +111,49 @@ const renderMenu = (menuItem) => {
             `}
           />
 
+          <${Expandable}
+            toggle=${'Legend'}
+            detail=${html`
+              <p>The map shows two parameters for territories:</p>
+              <ol>
+              <li>If the country is under any type of lockdown</li>
+              <li>If there are covid cases in the country</li>
+              </ol>
+            <table class=${styles}>
+<tr>
+<th></th>
+<th>Lockdown</th>
+<th>COVID-19</th>
+</tr>
+<tr>
+<td><div class="blue circle"/></td>
+<td>${close}</td>
+<td>${close}</td>
+</tr>
+<tr>
+<td><div class="green circle"/></td>
+<td>${close}</td>
+<td>${trues}</td>
+</tr>
+<tr>
+<td><div class="yellow circle"/></td>
+<td>${trues}</td>
+<td>${close}</td>
+</tr>
+<tr>
+<td><div class="red circle"/></td>
+<td>${trues}</td>
+<td>${trues}</td>
+</tr>
+<tr>
+<td><div class="default circle"/></td>
+<td>No Data <p class="note">(please contribute)</p></td>
```

Comment:
Maybe we can add a link to https://docs.google.com/forms/d/e/1FAIpQLSfDWe2qlzUnd3e-YYspMzT9adUswDEYIdJMb7jz7ule34-yiA/viewform here to make it easy for people to contribute